kafka/functional_tests/integrational_tests/static_config.yaml

```diff
@@ -48,6 +49,7 @@ components_manager:
         # /// [Kafka service sample - producer static config]
         # yaml
         kafka-producer-first:
+            client_id: test-client
```

Comment:
Add this to kafka-producer-second and delete from kafka-consumer and kafka-producer-first.
This option has default value and should not be typically used by user.

Reply:
Fixed